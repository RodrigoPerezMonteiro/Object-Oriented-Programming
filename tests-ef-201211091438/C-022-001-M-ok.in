5
4
yoda@jedi.edu
DISCOUNT
4
vader@imperial.net
DISCOUNT
4
yoda@jedi.edu
NOVELTY
4
kirk@starfleet.net
NOVELTY
4
all@all.edu
DISCOUNT
4
all@all.edu
NOVELTY
1
0
7
3
All Pizza
4
salada mista
4
Bifanas
0
0
5
6
yoda@jedi.edu
6
vader@imperial.net
6
kirk@starfleet.net
6
all@all.edu
0
0
