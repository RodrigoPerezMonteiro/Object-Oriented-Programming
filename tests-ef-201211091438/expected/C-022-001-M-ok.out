Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Tipo de alerta pretendido (DISCOUNT ou NOVELTY): Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Tipo de alerta pretendido (DISCOUNT ou NOVELTY): Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Tipo de alerta pretendido (DISCOUNT ou NOVELTY): Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Tipo de alerta pretendido (DISCOUNT ou NOVELTY): Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Tipo de alerta pretendido (DISCOUNT ou NOVELTY): Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Tipo de alerta pretendido (DISCOUNT ou NOVELTY): Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: OMNIVOROUS|Eat Everything|all@all.edu|0|0|0
VEGETARIAN|James T. Kirk|kirk@starfleet.net|0|0|0
OMNIVOROUS|Darth Vader|vader@imperial.net|0|0|0
VEGETARIAN|Master Yoda|yoda@jedi.edu|0|0|0
Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Identificador do restaurante (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): NOVELTY|Está disponível um novo prato do dia: +++salada mista+++. Não quer experimentar hoje?
Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): NOVELTY|Está disponível um novo prato do dia: +++salada mista+++. Não quer experimentar hoje?
Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): NOVELTY|Está disponível um novo prato do dia: +++salada mista+++. Não quer experimentar hoje?
NOVELTY|Está disponível um novo prato do dia: +++Bifanas+++. Não quer experimentar hoje?
Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 